api/db/seeds.rb

```diff
@@ -3,65 +3,150 @@
+        create: '/sessions/:session_id/positions',
+        update: '/positions/:id',
+        file: 'position.json',
+        unique: [:position_code, :session_id],
+        foreign_keys: [
+            {
+                table: Session,
+                key: :session_id
+            }
+        ]
+    },
+    {
+        table: Instructor,
+        create: '/instructors',
+        update: '/instructors/:id',
+        file: 'instructor.json',
+        unique: [:utorid],
+        foreign_keys: [
+            {
+                table: Position,
+                key: :position_ids,
+            }
+        ]
+    },
+    {
+        table: Applicant,
+        create: '/applicants',
+        update: '/applicants/:id',
+        file: 'applicant.json',
+        unique: [:utorid, :student_number],
+        foreign_keys: []
+    },
+    {
+        table: Application,
+        create: '/sessions/:session_id/applications',
+        update: '/applications/:id',
+        file: 'application.json',
+        unique: {
+            table: ApplicantDataForMatching,
+            child_keys: [:applicant_id],
+            parent_keys: [:session_id],
+            id: :application_id
+        },
+        foreign_keys: [
+            {
+                table: Session,
+                key: :session_id
+            },
+            {
+                table: Applicant,
+                key: :applicant_id
+            }
+        ]
+    },
+    {
+        table: PositionPreference,
+        create: '/applications/:application_id/add_preference',
+        update: '/position_preferences/:id',
+        file: 'position_preference.json',
+        unique: [:application_id, :position_id],
+        foreign_keys: [
+            {
+                table: Position,
+                key: :position_id,
+            },
+            {
+                table: Application,
+                key: :application_id,
+            }
+        ]
+    },
+    {
+        table: Assignment,
+        create: '/positions/:position_id/assignments',
+        update: '/assignments/:id',
+        file: 'assignment.json',
+        unique: [:application_id, :position_id],
+        foreign_keys: [
+            {
+                table: Position,
+                key: :position_id,
+            },
+            {
+                table: Applicant,
+                key: :applicant_id,
+            }
+        ]
+    },
+    {
+        table: WageChunk,
+        create: '/assignments/:assignment_id/add_wage_chunk',
+        update: '/wage_chunks/:id',
+        file: 'wage_chunk.json',
+        unique: [:assignment_id],
+        foreign_keys: [
+            {
+                table: Assignment,
+                key: :assignment_id,
+            }
+        ]
+    },
+    {
+        table: ReportingTag,
+        create: '/wage_chunks/:wage_chunk_id/add_reporting_tag',
+        update: '/reporting_tags/:id',
+        file: 'reporting_tag.json',
+        unique: [:wage_chunk_id, :position_id],
+        foreign_keys: [
```

Comment:
Why do you need to know anything about foreign keys?

Reply:
I actually used it to get the actually id value like in the discussion below.